docker/generic/start_proxy.py

```diff
@@ -464,7 +464,7 @@ def make_argparser():
         Enables a variety of debug features in both Config Manager and Envoy, such as:
         - Debug level per-request application logs in Envoy
         - Debug level service configuration logs in Config Manager
-        - Admin interface in Envoy
+        - Debug HTTP response headers
```

Comment:
do we not adding Admin port with enable_debug=true?   I think we should

Reply:
We were previously, but @JLXIA removed this in #67. I just updated the docs to match this.

One difficulty to add this behavior back is: we need to define a default port to enable admin interface (instead of user specifying port).

Reply:
hmm,  maybe in a separate PR to allow user to enable admin page. For GKE, it will be useful for some users